gnomad/resources/grch38/README.md

```diff
@@ -0,0 +1,49 @@
+# gnomAD Methylation Sites Data
+
+This directory contains methylation site annotations for the GRCh38 reference genome, used in gnomAD constraint calculations and variant annotation pipelines.
+
+## Files Description
+
+### BED Files
+- **methylation.bed**: Methylation site annotations for autosomes (chr1-22)
+- **methylation_chrX.bed**: Methylation site annotations for chromosome X
+
+### Hail Table Files
+- **methylation.ht/**: Hail Table containing methylation annotations for autosomes only
```

Comment:
keeping the slash hlps reinforce that HTs are actually directories rather than single files, but they do seem a little distracting (maybe just me?). what do you think about keeping or removing them?